index.js

```diff
@@ -3,7 +3,7 @@ var media = require('render-media')
 
 var raw = require('./lib/raw')
 var csv = require('./lib/csv')
-var geojson = require('./lib/geojson')
+var geojson = (typeof window !== 'undefined') ? require('./lib/geojson') : null
```

Comment:
no need to `( )` this, even simpler: `var geojson = typeof window !== 'undefined' && require('./lib/geojson')`